sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -91,4 +93,21 @@ export class StudentController {
     // Save student
     return this.service.createStudent(userToken, payload);
   }
+
+  @Patch("/sync")
+  async synchronizeFromUserInfo(
+    @UserToken() userToken: IUserToken,
+  ): Promise<UserSyncInfoDto> {
```

Comment:
Read that patch shouldnt return body so please remove the return UserSyncInfoDto Promise

Reply:
Just kept it because after our last conversation about it you had a strong position to do it :D
I will be happy to remove it 👍